src/SFA.DAS.FindApprenticeshipTraining.Api.UnitTests/Models/WhenBuildingGetProviderFeedbackAttributes.cs

```diff
@@ -133,105 +132,91 @@ public void Then_If_Same_Feedback_Attribute_Score_Then_The_Attribute_With_Most_R
             
             var response = new GetTrainingCourseProviderListItem().Map(source, sectorSubjectArea,1,new List<DeliveryModeType>(), new List<FeedbackRatingType>(), true);
             
-            response.Feedback.FeedbackAttributes.Weaknesses.Should().ContainInOrder(new List<string>{"Yattribute", "Zattribute", "Attribute"});
+            response.Feedback.FeedbackAttributes.FeedbackAttributeDetail.Select(x => x.AttributeName).Should().ContainInOrder(new List<string>{"Yattribute", "Zattribute", "Attribute"});
         }
         
         [Test, AutoData]
-        public void Then_If_Same_Feedback_Attribute_Score_And_Same_Responses_Then_The_Attribute_Is_Returned_In_Alphabetical_Order(GetProvidersListItem source, string sectorSubjectArea)
-        {
-            source.FeedbackAttributes = new List<GetFeedbackAttributeItem>
-            {
-                new GetFeedbackAttributeItem
-                {
-                    AttributeName = "Sixth Attribute",
-                    Strength = 14,
-                    Weakness = 13
-                },
-                new GetFeedbackAttributeItem
-                {
-                    AttributeName = "Seventh Attribute",
-                    Strength = 15,
-                    Weakness = 14
-                }
-            };
-            
-            var response = new GetTrainingCourseProviderListItem().Map(source, sectorSubjectArea,1,new List<DeliveryModeType>(),new List<FeedbackRatingType>(), true);
-            
-            response.Feedback.FeedbackAttributes.Strengths.Should().ContainInOrder(new List<string>{"Seventh Attribute"});
-        }
-        
-
-        [Test, AutoData]
-        public void Then_Returns_The_Top_Three_Feedback_Attribute_Strengths_And_Weaknesses(GetProvidersListItem source, string sectorSubjectArea)
+        public void Then_Returns_All_Available_Feedback_Attribute_Strengths_And_Weaknesses(GetProvidersListItem source, string sectorSubjectArea)
         {
             source.FeedbackAttributes = new List<GetFeedbackAttributeItem>
             {
                 new GetFeedbackAttributeItem
                 {
                     AttributeName = "First Attribute",
-                    Strength = 11,
-                    Weakness = 10
+                    Strength = 1,
```

Comment:
I get what you've done here in terms of equalising the strengths / weaknesses, and it returns all of them.

I don't know if we enforce ordering on the items returned from the API? IE ordered by Total votes ( strength / weakness ) then by strengths / then by weaknesses, which would then allow this test to prove something, as currently it just returns the list with the same values, so it feels like the test itself has become somewhat invalidated.

Reply:
Updated test